packages/ui/src/bounty/modals/AnnounceWorkEntryModal/AnnounceWorkEntryModal.tsx

```diff
@@ -52,21 +52,17 @@ export const AnnounceWorkEntryModal = () => {
     showModal,
   } = useModal<BountyAnnounceWorkEntryModalCall>()
   const { api, isConnected } = useApi()
-  const minWorkEntrantStake = api?.consts.bounty.minWorkEntrantStake.toNumber() ?? 0
   const { active: activeMember } = useMyMemberships()
   const { allAccounts } = useMyAccounts()
   const balances = useMyBalances()
-  const [amount, setAmount] = useState<string>(String(minWorkEntrantStake))
+  const minWorkEntrantStake = bounty.entrantStake
+  const [amount, setAmount] = useState<string>(minWorkEntrantStake.toString())
```

Comment:
```suggestion
  const amount = bounty.entrantStake
```
I think this `amount` should also replace `contribution`

Reply:
`Amount` is string and `contribution` is BN (as required for components it is passed to), so this 'conversion' is needed, but we can get rid of useMemo, as it won't change, do you agree?